components/leaderboard/CountdownTimer.tsx

```diff
@@ -3,18 +3,21 @@ import { CustomBox } from 'components/OffsetBorder'
 import { intervalToDuration } from 'date-fns'
 import type { Duration } from 'date-fns'
 
-import { nextMondayFrom } from 'utils/date'
 import { formatEventDate } from 'utils/events'
 
 const customFormatDuration = (x: Duration) =>
   `${x.days ? x.days + 'd : ' : ''}${x.hours ? x.hours + 'hr : ' : ''}${
     x.minutes ? x.minutes + 'min : ' : ''
   }${x.seconds ? x.seconds + 'sec' : '0sec'}`
 
-const CountdownTimer = () => {
+type CountdownTimerProps = {
+  end: Date
+  event: string
+}
+const CountdownTimer = (props: CountdownTimerProps) => {
+  const { end, event } = props
   const now = new Date()
   const [$time, $setTime] = useState(now)
```

Comment:
Repurposed this countdown timer since it wasn't used else where, also made it more generalizable.